tests/test_variants.py

```diff
@@ -151,21 +151,21 @@ def test_no_satisfiable_variants_raises_error():
 
 def test_zip_fields():
     """Zipping keys together allows people to tie different versions as sets of combinations."""
-    v = {'python': ['2.7', '3.5'], 'vc': ['9', '14'], 'zip_keys': [('python', 'vc')]}
+    v = {'python': ['3.7', '3.10'], 'vc': ['9', '14'], 'zip_keys': [('python', 'vc')]}
     ld = variants.dict_of_lists_to_list_of_dicts(v)
     assert len(ld) == 2
-    assert ld[0]['python'] == '2.7'
+    assert ld[0]['python'] == '3.7'
     assert ld[0]['vc'] == '9'
-    assert ld[1]['python'] == '3.5'
+    assert ld[1]['python'] == '3.10'
     assert ld[1]['vc'] == '14'
```

Comment:
The actual package used here isn't important so let's abstract it. I also suggest changing the imports to explicitly import stuff from the `conda_build.variants` (instead of importing it as `variants`):

```suggestion
    variants = {'packageA': ['1.2', '3.4'], 'packageB': ['5', '6'], 'zip_keys': [('pacakgeA', 'packageB')]}
    zipped = dict_of_lists_to_list_of_dicts(v)
    assert len(zipped) == 2
    assert zipped[0]['packageA'] == '1.2'
    assert zipped[0]['packageB'] == '5'
    assert zipped[1]['packageA'] == '3.4'
    assert zipped[1]['packageB'] == '6'
```

Reply:
It looks like this is being done in PR #4714 , so I'll leave this bit alone

Reply:
this wasn't actually done in the other PR